src/components/Modals/DeleteClientModal.jsx

```diff
@@ -32,12 +33,15 @@ const DeleteClientModal = ({
 }) => {
   const { state, dispatch } = useStatusNotification();
   const { removeUser } = useContext(UserListContext);
+  const [showSpinner, setShowSpinner] = useState(false);
 
   // Event handler for deleting client from client list
   const handleDeleteClient = async (event) => {
     event.preventDefault();
+    setShowSpinner(true);
     runNotification(
       `Deleting "${selectedClientToDelete?.person}" from client list...`,
+      'success',
```

Comment:
may be best to save `success` for when the action was actually successful (line 55)? and maybe these (when it's trying to do what the user asked) are better suited for either `warning` or `info` colors?